build.gradle

```diff
@@ -78,6 +78,15 @@ dependencies {
 
     // Monologue
     implementation 'com.github.shueja:Monologue:+'
+
+    // Gson
+    // Remove this once choreolib changes get upstreamed
+    implementation 'com.google.code.gson:gson:2.10.1'
```

Comment:
not sure if this is necessary

Reply:
Pretty sure Choreo requires it for all (de)serialization and json file creation, which we will probably want to keep for the time being.